content/300-guides/050-database/830-prismas-type-system.mdx

```diff
@@ -0,0 +1,152 @@
+---
```

Comment:
I think we should rename the file to `prisma-type-system`, without the `s`.

Reply:
Done